libs/crypto/keys_manager.hpp

```diff
@@ -36,26 +36,13 @@ namespace iroha {
    public:
     virtual ~KeysManager() = default;
 
-    /**
-     * Create a new keypair and store it as is on disk
-     * @return false if create account failed
-     */
-    virtual bool createKeys() = 0;
-
     /**
      * Create keys a new keypair and store it encrypted on disk
-     * @param pass_phrase is a password for the keys
+     * @param pass_phrase is used for private key encryption. Optional, default
+     * value is an empty string.
      * @return false if create account failed
      */
-    virtual bool createKeys(const std::string &pass_phrase) = 0;
-
-    /**
-     * Load plain-text keys associated with the manager, then validate loaded
-     * keypair by signing and verifying signature of test message
-     * @return nullopt if no keypair found locally, or verification failure;
-     *         related keypair otherwise
-     */
-    virtual boost::optional<shared_model::crypto::Keypair> loadKeys() = 0;
+    virtual bool createKeys(const std::string &pass_phrase = "") = 0;
```

Comment:
Not sure if this is a good idea, same below
http://www.codingstandard.com/rule/8-3-3-do-not-use-default-arguments/
>Use of default arguments can make code maintenance and refactoring more difficult. Overloaded forwarding functions can be used instead without having to change existing function calls.

It should be possible to define a function in the interface, such as
```C++
bool createKeys() {
  return createKeys("");
}
```

Reply:
Thanks! Did not know about the preferred way with overloading rather than default arguments.